### DISCUSSION TITLE
Convert .cdr-files?

### DISCUSSION THREAD

**Discussion by jadalber** [arkypita/LaserGRBL#1388] [Q&A]
I've used GCC laser at my previous job and wonder if someone know a method to convert .cdr files for use with laserGRBL ?

Thanks

**Comment by StuartB4** [comment]
https://cloudconvert.com/cdr-converter

**Comment by regpye** [comment]
You can use CorelDraw to export in several formats.  I use it all the time to prepare accurate work in CorelDraw and then export it as a JPG so I can use in LaserGRBL. Particularly good for doing special texts and line drawings, pattern shapes and things of that nature.

**Comment by juliascot0** [comment]
If you also want to convert your CorelDRAW CDR files to different formats, then the **[BitRecover CDR Converter](https://www.bitrecover.com/cdr/converter/)** Tool is one of the best tools you can use. It will help you easily export CDR files into multiple formats. Many users frequently use this tool because they really like its advanced features. This software offers numerous advanced functions that make your task even easier. You can complete the conversion process without any data loss.

**Comment by Tribo-2dot0** [comment]
Open the CDR in Inkscape and save it as SVG.

**Comment by vishalaryson-del** [comment]
If you’re trying to use a CorelDRAW CDR design with LaserGRBL, the main challenge is converting the CDR into a format that LaserGRBL can work with while keeping the original design intact. LaserGRBL supports image formats and SVG vector import, so converting the CDR to a suitable output format first can make the workflow easier.
For this, **MacSonik CDR Converter** can be useful if you have CDR files that need to be converted into other commonly used formats. It can help avoid manually opening and exporting each file through CorelDRAW, especially when you have multiple CDR designs.
For laser engraving/cutting, I’d recommend checking the converted file in your LaserGRBL preview before running the job, particularly for detailed text, line drawings, and vector shapes. This helps confirm that the dimensions and design elements are coming through correctly.

**Reply by Tribo-2dot0** [reply]
It's a shame it's paid... there are free options like Zamzar, AsanConvert, CloudConvert or Convertio. In Inkscape, you still have control over editing.
